Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -483,6 +483,11 @@ private void InitializeServiceLocator()
 
         private void EnsureMixedRealityRequirements()
         {
+            if (MixedRealityPlayspace == null)
```

Comment:
Isn't this dynamically created?

Where is that created again, and what would make this null?

Reply:
Yes, that happens when you invoke the property. So the null check creates it. If the property returns null something has gone very, very wrong.

Reply:
Conceptually does the MixedRealityToolkit service container require having a MixedRealityPlayspace? Should we actual update components to handle cases where a MixedRealityPlayspace doesn't exist compared to auto generating it?

Reply:
MRTK assumes the existence of a camera, and the camera requires the playspace to function correctly. I think that's a reasonable assumption as long as we continue to make it explicit by including the camera profile in the main config profile.